src/PowerShellEditorServices/Services/DebugAdapter/Handlers/PsesSemanticTokens.cs

```diff
@@ -0,0 +1,144 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.PowerShell.EditorServices.Services;
+using Microsoft.PowerShell.EditorServices.Services.TextDocument;
+using System.Management.Automation.Language;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+using OmniSharp.Extensions.LanguageServer.Protocol.Document.Proposals;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models.Proposals;
+using System;
+using Microsoft.PowerShell.EditorServices.Utility;
+
+namespace Microsoft.PowerShell.EditorServices.Handlers
+{
+    //Disable warnings having to do with SemanticTokensHandler being labelled obsolete
```

Comment:
```suggestion
    // SemanticTokensHandler is labeled "Obsolete" because that is how Omnisharp marks proposed LSP features. Since we want this proposed feature, we disable this warning.
```

Reply:
Also, can you just put the `#pragma warning restore 618` on the line right after you use `SemanticTokensHandler` (23.5)? Does that still work?

Reply:
There's warnings for all the semantic related things in the OmniSharp.Extensions.LanguageServer.Protocol.Document.Proposals namespace like SemanticTokensDocument, SemanticTokenType and SemanticTokensBuilder

Reply:
Got it. Nevermind on that then.